src/pages/user/my-proposed-resources.tsx

```diff
@@ -1,26 +1,58 @@
 import Head from "next/head";
+import { useState, useMemo } from "react";
 
 import { PageLayout } from "@/components/page-layout";
 import { ResourceList } from "@/components/resource-list";
 import { api } from "@/utils/api";
+import { Button } from "@/components/ui/button";
+
+enum TABS {
```

Comment:
With the proposed `publicationStatus` flag, then we wouldn't need the tabs, and we would just show a different badge on each row.

Reply:
Personally, I would like to be able to have them separate, at least on this page. I will change it if you really dont like tabs though

Reply:
Thats fine, Im not that attached to the tabs